packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -81,94 +124,100 @@
             :class="[$style['c-accountInfo-changePasswordButton']]"
             data-test-id="account-info-submit-button"
             button-type="secondary"
+            href="/change-password?returnurl=/account/info"
             button-size="large"
             is-full-width
             action-type="submit">
             {{ $t('buttons.changePassword') }}
         </f-button>
 
-        <hr>
-
-        <p>{{ $t('deleteAccountMessage') }}</p>
-
-        <f-link
-            is-distinct
-            href="/help"
-            target="_blank">
-            {{ $t('deleteAccountLink') }}
-        </f-link>
-    </card>
+        <delete-account />
+    </f-card>
 </template>
 
 <script>
 import { VueGlobalisationMixin } from '@justeat/f-globalisation';
 
-import Card from '@justeat/f-card';
+import FErrorMessage from '@justeat/f-error-message';
+import '@justeat/f-error-message/dist/f-error-message.css';
+
+import FCard from '@justeat/f-card';
 import '@justeat/f-card/dist/f-card.css';
 
 import FormField from '@justeat/f-form-field';
 import '@justeat/f-form-field/dist/f-form-field.css';
 
-import FLink from '@justeat/f-link';
-import '@justeat/f-link/dist/f-link.css';
-
 import FButton from '@justeat/f-button';
 import '@justeat/f-button/dist/f-button.css';
 
+import '@justeat/f-link/dist/f-link.css';
```

Comment:
Do you still need this if you've got rid of the link component?

Reply:
I register the CSS here so that I don't need to register it twice in the sub components that use it

Reply:
Could you add a comment above this import to say that the styles are included here, but the component itself is included as part of linked child components?